metadata.md

```diff
@@ -28,58 +38,87 @@ There may also be any number of delegated target roles.
 
 Signed by: Root role.
 
-Specifies the other top-level roles. When specifying these roles, the trusted keys for each role are listed along with the minimum number of those keys which are required to sign the role's metadata. We call this number the signature threshold.
+Specifies the other top-level roles. When specifying these roles, the trusted
+keys for each are listed, along with the minimum number of those keys required
+to sign the role's metadata. We call this number the signature threshold.
 
 See [example](https://raw.githubusercontent.com/theupdateframework/tuf/develop/tests/repository_data/repository/metadata/root.json) of Root metadata.
 
 ## Targets Metadata (targets.json)
 
 Signed by: Targets role.
 
-The targets.json metadata file lists hashes and sizes of target files. Target files are the actual files that clients are intending to download (for example, the software updates they are trying to obtain).
+Target files are the actual files that client wants to download, such as
+the software updates they are trying to obtain. The targets.json metadata
+file lists hashes and sizes of target files.
 
-This file can optionally define other roles to which it delegates trust. Delegating trust means that the delegated role is trusted for some or all of the target files available from the repository. When delegated roles are specified, they are specified in a similar way to how the Root role specifies the top-level roles: the trusted keys and signature threshold for each role is given. Additionally, one or more patterns are specified which indicate the target file paths for which clients should trust each delegated role.
+This file can optionally define other roles to which it delegates trust,
+or specifies that another role is trusted for some or all of the target files
+available from the repository. When delegated roles are specified, it is done
+so in a way similar to how the Root role specifies the top-level roles: by giving
+the trusted keys and signature threshold for each role. Additionally, one or more
+patterns are specified to indicate the target file paths for which clients
```

Comment:
Instead of "patterns," should we say "glob patterns" (perhaps with a Wikipedia link), to make it clear what is meant?

Reply:
You're the expert here, so i'll take your word for it. I changed the phrase and added the Wikipedia link

Reply:
changed the phrase and added a wikipedia link